resources/js/app.js

```diff
@@ -566,15 +562,12 @@ function initAutocomplete() {
       type: 'post',
       url: '/party/remove-volunteer',
       data: {
-        user_id : user_id,
-        event_id : event_id
+        id : id,
       },
       datatype: 'json',
       success: function(json) {
         if( json.success ){
-          jQuery('.volunteer-' + user_id).fadeOut();
-          jQuery('#'+type+'-counter').text();
-          counter.text( current_count - 1 );
```

Comment:
Is this decremented some other way (or no longer needs decrementing?)

Reply:
This is old code which used to manually update counts on the main parent page from within the child modal.  But the main page is now in Vue, so it gets updated because the store changes.